settings.py

```diff
@@ -152,3 +160,4 @@
 
 # digest email css
 NOTIFICATION_DIGEST_EMAIL_CSS = 'edx_notifications/css/email_digests.css'
+MCKA_APP_URL = 'https://www.mckinseyacademy.com/'
```

Comment:
We probably should have a better way to determine the domain of the environment notifications app is running in.